src/Aspire.Hosting.Azure.Storage/AzureStorageExtensions.cs

```diff
@@ -126,12 +128,15 @@ public static IResourceBuilder<AzureStorageResource> RunAsEmulator(this IResourc
 
         builder.WithHealthCheck(healthCheckKey);
 
-        if (configureContainer != null)
-        {
-            var surrogate = new AzureStorageEmulatorResource(builder.Resource);
-            var surrogateBuilder = builder.ApplicationBuilder.CreateResourceBuilder(surrogate);
-            configureContainer(surrogateBuilder);
-        }
+        // The default arguments list is coming from https://github.com/Azure/Azurite/blob/c3f93445fbd8fd54d380eb265a5665166c460d2b/Dockerfile#L47C6-L47C106
+        // They need to be repeated in order to be able to add --skipApiVersionCheck
+
+        var surrogate = new AzureStorageEmulatorResource(builder.Resource);
+        var surrogateBuilder = builder.ApplicationBuilder
+            .CreateResourceBuilder(surrogate)
+            .WithArgs("azurite", "-l", "/data", "--blobHost", "0.0.0.0", "--queueHost", "0.0.0.0", "--tableHost", "0.0.0.0", SkipApiVersionCheckArgument);
```

Comment:
Don't you need to also specify the port?

Reply:
The ones we set are the defaults (10000, 10001, 10002). Do you prefer these as constants and set them on the command line too? Technically this is exactly the same as before this PR.

Reply:
Revised my comments. These are customizable.